### PR TITLE
[#244] Exclude some OTP applications from indexing

### LINKED ISSUE #244: Be able to exclude OTP applications from indexing
Some of the OTP applications are heavy and rarely used by end users. A notable example is represented by the [megaco](http://erlang.org/doc/man/megaco.html) application. We could introduce a default exclusion list for OTP applications that are not part of the indexing. It should be possible to configure this exclusion list via the `erlang_ls.config` file at project and (maybe, not necessary for this ticket) at installation level (e.g. via a `~/.config/erlang_ls` file or similar.

References:

https://bugs.erlang.org/browse/ERL-1097

### PR DESCRIPTION
**Pull request by jfacorro**
### Description

Configuration option to exclude OTP application from the indexing process.

Fixes #244.

### CODE REVIEW

File: src/els_config.erl [hunk 1]
```diff
@@ -23,6 +23,13 @@
 %%==============================================================================
 -define(DEFAULT_CONFIG_PATH, "erlang_ls.config").
 -define(SERVER, ?MODULE).
+-define( DEEFAULT_EXCLUDED_OTP_APPS
```
Comment (robertoaloi) [inline src/els_config.erl]: Typo: DEEFAULT

Comment (robertoaloi) [reply src/els_config.erl]: Also, please keep them in lexicographical order.

File: src/els_config.erl [hunk 2]
```diff
@@ -56,9 +65,17 @@ initialize(RootUri, Capabilities, InitOptions) ->
...
                                         ])),
-  OtpPath = maps:get("otp_path", Config, code:root_dir()),
-  DepsDirs = maps:get("deps_dirs", Config, []),
-  IncludeDirs = maps:get("include_dirs", Config, ["include"]),
+  OtpPath        = maps:get("otp_path", Config, code:root_dir()),
+  DepsDirs       = maps:get("deps_dirs", Config, []),
+  IncludeDirs    = maps:get("include_dirs", Config, ["include"]),
+  OtpAppsExclude = maps:get( "otp_apps_exclude"
+                           , Config
+                           , ?DEEFAULT_EXCLUDED_OTP_APPS
```
Comment (robertoaloi) [inline src/els_config.erl]: And same typo here.

File: src/els_config.erl (hunk 1 above)
Comment (jfacorro) [reply src/els_config.erl]: 🤦‍♂ Fixed

**Pull request merged**